tests/tests.cpp

```diff
@@ -293,6 +295,38 @@ void test_future ()
   test_future_get_value(async_member);
 }
 
+void test_latch ()
+{
+  std::latch latch(5);
+
+  std::thread thread1([&latch] { latch.wait(); });
+  std::thread thread2([&latch] { latch.arrive_and_wait(2); });
+
+  latch.count_down();
+
+  log("Testing count_down...");
+  latch.count_down(2);
+
+  thread1.join();
+  thread2.join();
+}
+
```

Comment:
You can test for `noexcept` specifiers using `static_assert(noexcept(expression), "message")`. In the case of `latch`, `max()` and `try_wait()` are both `noexcept`.